bindings/jni/msvc/vs2010/libczmqjni/libczmqjni.props

```diff
@@ -32,7 +32,6 @@
     <ClCompile>
       <AdditionalIncludeDirectories>..\..\..\..\..\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
       <AdditionalIncludeDirectories>$(JAVA_HOME)\include;$(JAVA_HOME)\include\win32;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
-      <AdditionalIncludeDirectories>..\..\..\..\..\..\libzmq\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>

```

Comment:
this shouldn't happen right? Do you have libzmq cloned at the same level as czmq?